lib/server.ml

```diff
@@ -108,8 +113,8 @@ let connect channel ?offer () =
     let rec negotiate_tls () =
       read_hdr_and_payload channel.read_clear
       >>= fun (opt, _) -> match opt with
-      | Option.ExportName -> fail (Failure "Client requested export over cleartext channel but server is in FORCEDTLS mode.")
-      | Option.Abort -> fail (Failure "Client requested abort (before negotiating TLS).")
+      | Option.ExportName -> fail_with "Client requested export over cleartext channel but server is in FORCEDTLS mode."
```

Comment:
please be explicit and use `Lwt.fail_with` (here and on the line below)

Reply:
Done - now only `Lwt.Infix` is opened.